draft-ietf-quic-recovery.md

```diff
@@ -1232,10 +1255,7 @@ OnAckReceived(ack, pn_space):
       IncludesAckEliciting(newly_acked_packets)):
     latest_rtt =
       now() - newly_acked_packets.largest().time_sent
-    ack_delay = 0
-    if (pn_space == ApplicationData):
-      ack_delay = ack.ack_delay
-    UpdateRtt(ack_delay)
+    UpdateRtt(ack.ack_delay)
```

Comment:
We should change line ~1285 down there vv to:

```
  if handshake confirmed:
    ack_delay = min(ack_delay, max_ack_delay)
```

Reply:
GetPtoTimeAndSpace needs to be updated to use handshake confirmed as well

Reply:
Good catch.

Reply:
Done.